src/site/apt/tutorial.apt

```diff
@@ -169,7 +169,7 @@ executor.execute(cmdLine, resultHandler);
 
 // some time later the result handler callback was invoked so we
 // can safely request the exit value
-int exitValue = resultHandler.waitFor();
+resultHandler.waitFor();
```

Comment:
I would also show the exitValue = resultHandler.getExitValue(); call as the comment suggests.

Reply:
Hi @ecki,
Does the PR look ok to you now?

Reply:
Looks good - maybe in the future we should use snippets or at least have a copy in a test class.

you might need to add the changes.xml entry?

Reply:
@ecki updated changes.xml

Reply:
In general, don't update changes.xml, if every PR did that, they would all create merge problems.